firmware/library/L1_Drivers/uart.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+class UARTInterface
```

Comment:
See the naming rules for classes here: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules